### ISSUE TITLE
detekt issues in generated code for samples-glean

### ISSUE THREAD

**Issue by pocmo** [mozilla-mobile/android-components#1423]
Every now and then this shows up:

``` 
Ruleset: comments
Ruleset: complexity
Ruleset: empty-blocks
Ruleset: exceptions
Ruleset: naming - 15min debt
        PackageNaming - [kotlin.jinja2.kt] at /Users/skaspari/Projects/Mozilla/android-components/samples/glean/build/bootstrap/Miniconda3/lib/python3.7/site-packages/glean_parser/templates/kotlin.jinja2.kt:9:1
        PackageNaming - [BrowserEngagement.kt] at /Users/skaspari/Projects/Mozilla/android-components/samples/glean/build/telemetry/src/debug/kotlin/BrowserEngagement.kt:9:1
        PackageNaming - [Basic.kt] at /Users/skaspari/Projects/Mozilla/android-components/samples/glean/build/telemetry/src/debug/kotlin/Basic.kt:9:1
Ruleset: performance
Ruleset: potential-bugs
Ruleset: style - 5min debt
        SpacingBetweenPackageAndImports - [kotlin.jinja2.kt] at /Users/skaspari/Projects/Mozilla/android-components/samples/glean/build/bootstrap/Miniconda3/lib/python3.7/site-packages/glean_parser/templates/kotlin.jinja2.kt:11:49
```

**Comment by pocmo** [comment]
This broke our 0.32.0 release now :(

**Comment by jonalmeida** [comment]
Related issue: https://github.com/mozilla-mobile/android-components/issues/1193

**Comment by pocmo** [comment]
@mdboom @Dexterp37 I'm going to disable detekt for /build/ folders since that is not code that we write ourselves. However other apps using `glean` may see that too.

**Comment by mdboom** [comment]
See https://github.com/mozilla/glean_parser/pull/20 for a fix.

**Comment by Dexterp37** [comment]
> See [mozilla/glean_parser#20](https://github.com/mozilla/glean_parser/pull/20) for a fix.

@pocmo @mdboom if the fix works as expected, should we revert 2648ff2a0a6e17d6e92c600fad199a17269bd9f9 ?

**Comment by pocmo** [comment]
I think we still want to avoid running detekt/ktlint on files in /build/ since this is not code written by us.
